src/Tests/Interop/Interop.idl

```diff
@@ -0,0 +1,154 @@
+// Modern IDL 3.0: https://docs.microsoft.com/en-us/uwp/midl-3/intro
+
+namespace Windows.Interop
+{
+    // accountssettingspaneinterop.idl
+   [uuid(D3EE12AD-3865-4362-9746-B75A682DF0E6)]
+   interface IAccountsSettingsPaneInterop 
+   {
+       Object GetForWindow(
+           UInt64 appWindow,
```

Comment:
I think all of these should be "IntPtr hwnd" instead of "UInt64 appWindow"

Reply:
IntPtr is not an IDL type - it's a C# type that requires a lossless IDL representation

Reply:
This speaks to the challenges of trying to express non-WinRT APIs in MIDL3. `IntPtr` isn't available, but `UInt64` is not the right type either. This is an `HWND`, which is either 64 bits or 32 bits depending on architecture. You would need to define distinct x86 and x64 variants of this interface in MIDL3 or just use [win32metadata](https://github.com/microsoft/win32metadata/).

Reply:
I suspected the HWND modeling was incorrect - thanks

Reply:
The Interop interfaces are C++ interfaces, often deriving from IUnknown, not MIDL interfaces deriving from IInspectable. IntPtr is the correct data type because the real C++ interfaces use HWND, which is pointer size.

Reply:
Update: added custom mapping of WinRT.Interop.HWND to System.IntPtr